flink-kubernetes/pom.xml

```diff
@@ -0,0 +1,208 @@
+			<version>${kubernetes.client.version}</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-runtime_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+			<type>test-jar</type>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-clients_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+			<type>test-jar</type>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-test-utils_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+			<scope>test</scope>
+		</dependency>
+
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>shade-flink</id>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration combine.children="append">
+							<artifactSet>
+								<includes combine.children="append">
+									<include>io.fabric8:kubernetes-client</include>
+									<include>io.fabric8:kubernetes-model</include>
+									<include>io.fabric8:kubernetes-model-common</include>
+
+									<!-- Shade all the dependencies of kubernetes client  -->
+									<include>com.fasterxml.jackson.core:jackson-core</include>
+									<include>com.fasterxml.jackson.core:jackson-annotations</include>
+									<include>com.fasterxml.jackson.core:jackson-databind</include>
+									<include>com.fasterxml.jackson.dataformat:jackson-dataformat-yaml</include>
+									<include>com.squareup.okhttp3:*</include>
+									<include>com.squareup.okio:okio</include>
+									<include>org.yaml:*</include>
+									<include>com.mifmif:*</include>
+									<include>dk.brics.automaton:*</include>
+								</includes>
+							</artifactSet>
+							<filters combine.children="append">
+								<filter>
+									<artifact>*:*</artifact>
+									<excludes>
+										<exclude>*.aut</exclude>
+										<exclude>META-INF/maven/**</exclude>
+										<exclude>META-INF/services/**</exclude>
+										<exclude>META-INF/proguard/**</exclude>
+										<exclude>OSGI-INF/**</exclude>
+										<exclude>schema/**</exclude>
+										<exclude>*.vm</exclude>
+										<exclude>*.properties</exclude>
+										<exclude>*.xml</exclude>
+										<exclude>META-INF/jandex.idx</exclude>
+										<exclude>license.header</exclude>
+									</excludes>
+								</filter>
+							</filters>
+							<relocations>
+								<relocation>
+									<pattern>com.fasterxml.jackson</pattern>
+									<shadedPattern>org.apache.flink.kubernetes.shadded.com.fasterxml.jackson</shadedPattern>
+								</relocation>
+								<relocation>
+									<pattern>okhttp3</pattern>
```

Comment:
Why not to shade `com.squareup.okhttp3`?

Reply:
It has been shaded.
`
<relocation>
  <pattern>okhttp3</pattern>					 
  <shadedPattern>org.apache.flink.kubernetes.shadded.okhttp3</shadedPattern>
</relocation>
`

Reply:
I mean it there any concern or blocker to directly shade `com.squareup.okhttp3` so that we have to shade the keyword `okhttp3`? 
That's how it is doing with other artifacts.

Reply:
Aha, i think the package name does not contain the artifact name. The package name is `okhttp3`,so the relocation is `org.apache.flink.kubernetes.shadded.okhttp3`. You could check the content of `okhttp-3.12.jar` to confirm that.

Reply:
I see. Thanks for the explanation!

Reply:
+1. shaded is fine actually. sorry for missing the line in the shade plugin